demo/src/main/java/com/jira/demo/service/SquadService.java

```diff
@@ -10,9 +10,11 @@
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+
```

Comment:
Don't forget to format your code with combination of `Ctrl + Alt + L` to keep your code clean and consistent